docs/build/tutorials/platform/subnets/create-a-virtual-machine-vm.md

```diff
@@ -1480,7 +1480,7 @@ cp -n <path to your binary> $GOPATH/src/github.com/ava-labs/avalanchego/build/pl
 
 #### Node is not running
 If your node isn't running yet, you can install all virtual machines under your `plugin` directory by starting the node.
-See [here](../tutorials/nodes-and-staking/run-avalanche-node/#start-a-node-and-connect-to-avalanche) for references on
+See [here](../../nodes-and-staking/run-avalanche-node.md#start-a-node-and-connect-to-avalanche) for references on
```

Comment:
Hm it's probably best to not direct them how to start their node. The page is swarmed with other information like wallet etc. Can we remove this line and just tell it's sufficient to copy binaries before startup? sorry for the nit 😶

Reply:
Sure : )